### DISCUSSION TITLE
Power off does not work

### DISCUSSION THREAD

**Discussion by tmesza** [MiczFlor/RPi-Jukebox-RFID#542] [Q&A] [answered]
Hello together,
i want to ask if somebody can help me with my power off issue?
I build the Jukebox with the instructions from MiczFlor and a Pi3B+, thanks for sharing that great work. The kids are like the box :)
Sometimes is it not possible to power off the box by using the on/off button, the red led on the RPI is showing a red light. I use an on/off shim. I can only power off the RPI by remove the Power by the USB-Power connection.
thanks in advance for your help.
tmesza

**Comment by domu83** [comment]
same problem on my setup as well, no resolution found so far

**Comment by RogoTe** [comment]
Might be a problem with your onoff-shim. The problem is that without such shim you cannot power down a RaspPi completly. The red LED stayes on although the SoC is down.
So my best guess is, that the shim somehow did not do it's job.

**Comment by s-martin** [comment, marked as answer]
@splitti provided an extensive explanation regarding onoffshim in [his blog](http://splittscheid.de/selfmade-phoniebox/#3A), but it’s in German.
